CHANGELOG.next.asciidoc

```diff
@@ -71,9 +71,8 @@ https://github.com/elastic/beats/compare/v8.8.1\...main[Check the HEAD diff]
 - Upgraded apache arrow library used in x-pack/libbeat/reader/parquet from v11 to v12.0.1 in order to fix cross-compilation issues {pull}35640[35640]
 - Fix panic when MaxRetryInterval is specified, but RetryInterval is not {pull}35820[35820]
 - Do not print context cancelled error message when running under agent {pull}36006[36006]
-
-
 - Fix recovering from invalid output configuration when running under Elastic-Agent {pull}36016[36016]
+- Improve StreamBuf append {pull}35928[35928]
```

Comment:
```suggestion
- Improve StreamBuf append to improve performance when reading long lines from files. {pull}35928[35928]
```

Let's add a just a little bit more context to that user's know what the benefit to them is.